jujugui/static/gui/src/app/components/entity-files/_entity-files.scss

```diff
@@ -0,0 +1,15 @@
+.entity-files {
+    ul {
```

Comment:
While one of the benefits is markup changes I think that's secondary to the removal of the cascade effect. Because of these styles if there were any sub components which implemented a ul/li they would inherit these styles. I've spent so much time overwriting styles that are much to generic that I always try to er on the side of caution.